packages/@react-aria/actiongroup/src/useActionGroup.ts

```diff
@@ -66,15 +56,15 @@ export function useActionGroup<T>(props: ActionGroupProps<T>, state: ActionGroup
   });
   let tabIndex = isFocusWithin ? -1 : 0;
 
+  let role = BUTTON_GROUP_ROLES[selectionMode];
```

Comment:
shouldn't this be off in an aria hook?
looks like there are a lot of things in this file that should be in hooks?